xml/ay_security_settings.xml

```diff
@@ -127,12 +128,35 @@
    Set the minimum and maximum possible user and group IDs.
   </para>
  </sect2>
-
  <sect2 xml:id="CreateProfile-Security-selinux">
-  <title>SELinux settings</title>
+  <title>Linux Security Module (LSM) settings</title>
   <para>
-   Configuring SELinux mode. Possible values are
-   <literal>permissive,enforcing</literal> and <literal>disabled</literal>.
+    In &sle;&nbsp;15.4 and up, the installation control file has a new
+    option, <literal>&lt;lsm_select></literal> for configuring
```

Comment:
Just a question, is `&gt;`not needed?

```suggestion
    option, <literal>&lt;lsm_select&gt;</literal> for configuring
```

Reply:
It's not required, &lt; by itself is enough for it to build correctly. Probably good to be consistent.